package.json

```diff
@@ -122,7 +124,7 @@
     "test:smoke:components-js": "yarn --cwd ./packages/components-js test:smoke",
     "updateAlgolia": "yarn --cwd ./packages/storefront updateAlgolia",
     "clean": "yarn clean:node_modules && yarn clean:build",
-    "clean:node_modules": "rm -rf ./node_modules && rm -rf ./yarn-error.log && yarn clean:assets:node_modules && yarn clean:components:node_modules && yarn clean:components-angular:node_modules && yarn clean:components-js:node_modules && yarn clean:components-react:node_modules && yarn clean:components-vue:node_modules && yarn clean:email:node_modules && yarn clean:fonts:node_modules && yarn clean:icons:node_modules && yarn clean:marque:node_modules && yarn clean:meta-icons:node_modules && yarn clean:shared:node_modules && yarn clean:storefront:node_modules && yarn clean:utilities:node_modules && yarn clean:utilities-deprecated:node_modules",
+    "clean:node_modules": "rm -rf ./node_modules && rm -rf ./yarn-error.log && yarn clean:assets:node_modules && yarn clean:components:node_modules && yarn clean:components-angular:node_modules && yarn clean:components-js:node_modules && yarn clean:components-react:node_modules && yarn clean:components-vue:node_modules && yarn clean:crawler:node_modules && yarn clean:email:node_modules && yarn clean:fonts:node_modules && yarn clean:icons:node_modules && yarn clean:marque:node_modules && yarn clean:meta-icons:node_modules && yarn clean:shared:node_modules && yarn clean:storefront:node_modules && yarn clean:utilities:node_modules && yarn clean:utilities-deprecated:node_modules",
     "clean:build": "yarn clean:assets:build && yarn clean:components:build && yarn clean:components-angular:build && yarn clean:components-js:build && yarn clean:components-react:build && yar clean:components-vue:build && yarn clean:email:build && yarn clean:fonts:build && yarn clean:icons:build && yarn clean:marque:build && yarn clean:meta-icons:build && yarn clean:shared:build && yarn clean:storefront:build && yarn clean:utilities:build && yarn clean:utilities-deprecated:build",
```

Comment:
Is there nothing to clean regarding builds?

Reply:
@stephanschroeter We don't have any builds right now, so no builds to be removed.. Removing reports during clean is a good idea.. But if we do it - then we indeed don't need "removing outdated reports" functionality in crawler

Reply:
Added "remove reports" to clean